rules/opentitan_gdb_test.bzl

```diff
@@ -0,0 +1,122 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+load("@lowrisc_opentitan//rules:rv.bzl", "rv_rule")
+
+def _opentitan_gdb_fpga_cw310_test(ctx):
+    test_script = '''#!/usr/bin/env bash
+        set -e
+
+        COLOR_RED='\\x1b[0;31m'
+        COLOR_GREEN='\\x1b[0;32m'
+        COLOR_PURPLE='\\x1b[0;35m'
+
+        function prefix_lines() {{
+            LABEL="$1"
+            COLOR="$2"
+            COLOR_RESET='\\x1b[m'
+            sed -Eu "s/(.*)/$COLOR[$LABEL]$COLOR_RESET \\1/"
+        }}
+
+        set -x
+
+        ./opentitantool load-bitstream --rom-kind=TestRom rom.bit
```

Comment:
Hmm, we may need to update this line when we start running this in the CI. See [here](https://cs.opensource.google/opentitan/opentitan/+/master:rules/opentitan_test.bzl;drc=b05d61d943e1bfff005bfa5fa377e1e31d600d21;l=450).

Reply:
Interesting, there does seem to be a built-in heuristic for this in [sw/host/opentitanlib/src/transport/cw310/mod.rs](https://cs.opensource.google/opentitan/opentitan/+/master:sw/host/opentitanlib/src/transport/cw310/mod.rs;l=116;drc=e6b0661560a27c9826de3dcd0eae3c93390712b1). I'm not sure whether it's better to trust the heuristic or to explicitly state the `--cw310-uarts`. Maybe @cfrantz can advise :)